app/models/concerns/pxe_loader_support.rb

```diff
@@ -5,7 +5,7 @@ module PxeLoaderSupport
   PXE_KINDS = {
     :PXELinux => /^(pxelinux.*|PXELinux (BIOS|UEFI))$/,
     :PXEGrub => /^(grub\/|Grub UEFI).*/,
-    :PXEGrub2 => /^(grub2\/|Grub2 (BIOS|UEFI|ELF)|http.*grub2\/).*/,
```

Comment:
FYI: While fixing the tests I stumbled across this mapping of filename / loader name to template kind, because a test was broken due to the changes of the filenames down below. I found out that actually a mapping from filename to template kind is never used in the code (besides in the tests). There is exactly one usage of this mapping and there only the loader name is used. So I was wondering if there is a historic reason for the mapping by filename? (cc @lzap)

Reply:
If `git log -Gxxxx` does not explain it, then delete, yeah. I don’t remember to be honest.

Reply:
Ok, searching with `git log` reveals more places I would want to check before removing the filename mapping than I can cope with for now. Better to do this in a separate PR.